charts/consul/test/unit/connect-inject-deployment.bats

```diff
@@ -617,83 +575,36 @@ EOF
   [ "${actual}" = "key" ]
 }
 
-@test "connectInject/Deployment: Adds -tls-cert-dir=/etc/connect-injector/certs to command" {
+@test "connectInject/Deployment: CONSUL_HTTP_ADDR env variable is set to an https address when global.tls.enabled is true" {
   cd `chart_dir`
   local actual=$(helm template \
       -s templates/connect-inject-deployment.yaml  \
       --set 'connectInject.enabled=true' \
       --set 'global.tls.enabled=true' \
       . | tee /dev/stderr |
-      yq '.spec.template.spec.containers[0].command | any(contains("-tls-cert-dir=/etc/connect-injector/certs"))' | tee /dev/stderr)
-  [ "${actual}" != "" ]
+      yq -r '.spec.template.spec.containers[0].env[] | select(.name == "CONSUL_HTTP_ADDR").value' | tee /dev/stderr)
+  [ "${actual}" = "https://release-name-consul-server.default.svc:8501" ]
 }
 
-#--------------------------------------------------------------------
-# global.tls.enableAutoEncrypt
```

Comment:
Auto-encrypt is no longer supported for the connect-inject deployment as this is a setting used only when talking to consul clients.